modules/flowable-bpmn-converter/src/test/java/org/flowable/editor/language/xml/ParallelGatewayConverterTest.java

```diff
@@ -0,0 +1,57 @@
+/* Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.flowable.editor.language.xml;
+
+import static junit.framework.TestCase.assertFalse;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.tuple;
+import static org.junit.Assert.assertTrue;
+
+import java.util.List;
+
+import org.flowable.bpmn.model.BpmnModel;
+import org.flowable.bpmn.model.EventGateway;
+import org.flowable.bpmn.model.FlowElement;
+import org.flowable.bpmn.model.FlowableListener;
+import org.flowable.bpmn.model.ImplementationType;
+import org.flowable.bpmn.model.ParallelGateway;
+import org.junit.Test;
+
+/**
+ * Test for ACT-1657
+ * 
+ * @author Frederik Heremans
+ */
+public class ParallelGatewayConverterTest extends AbstractConverterTest {
+
+    @Test
+    public void convertXMLToModel() throws Exception {
+        BpmnModel bpmnModel = readXMLFile();
+        validateModel(bpmnModel);
+    }
+
+    @Override
+    protected String getResource() {
+        return "parallelgatewaymodel.bpmn";
+    }
+
+    private void validateModel(BpmnModel model) {
+        FlowElement flowElement = model.getMainProcess().getFlowElement("parallelGateway");
+        assertThat(flowElement).isNotNull();
+        assertThat(flowElement).isInstanceOf(ParallelGateway.class);
+
+        ParallelGateway gateway = (ParallelGateway) flowElement;
+        assertTrue(gateway.isAsynchronous());
+        assertFalse(gateway.isExclusive());
```

Comment:
Instead of mixing styles of assertions, why not this instead?

`assertThat(gateway.isAsynchronous()).isTrue();`
`assertThat(gateway.isExclusive()).isFalse()`